core/src/global/test/base-components.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { isBaseComponent, resetBaseComponentsCache } from '../base-components';
+import { Config } from '../config';
+
+describe('isBaseComponent()', () => {
```

Comment:
This will probably be work for another time, but I wonder if we should expand our Playwright screenshot testing to also include `base` styling?

Reply:
Both the MD/iOS stylesheets pull in the base stylesheet, so if there's a change to the structure of a component those changes should be reflected in the MD/iOS screenshots.

Reply:
Thinking about this more... it might be good to have a single screenshot of each component using the base styles.   Currently our screenshots can get fairly complex with functionality/interaction with other components, but I'd imagine these can be pretty simple screenshots of just the component.

Reply:
Yeah, if stuff like the solid black screen backdrop had happened as a separate regression down the line, a screenshot test would catch it while our existing test setup would miss it. I'll make a ticket.